tests/rbd_mirror/test_mirror_move_secondary_trash.py

```diff
@@ -0,0 +1,81 @@
+from tests.rbd.rbd_utils import Rbd
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_image_move_secondary_trash(rbd_mirror, pool_type, **kw):
+    """
+    Test to check secondary image move to trash is failed as image will be locked and busy.
+    Args:
+        rbd_mirror: rbd mirror object
+        pool_type: ec pool or rep pool
+        **kw:
+    Returns:
+        0 if test pass, else 1
+    """
+    try:
+        mirror1 = rbd_mirror.get("mirror1")
+        config = kw.get("config")
+        pool = config[pool_type]["pool"]
+        image = config[pool_type]["image"]
+        imagespec = pool + "/" + image
+
+        rbd1, rbd2 = [
+            Rbd(**kw, req_cname=cluster_name)
+            for cluster_name in kw.get("ceph_cluster_dict").keys()
+        ]
+
+        mirror1.benchwrite(imagespec=imagespec, io=config.get("io-total", "1G"))
+
+        # Move secondary image to trash
+        if rbd2.move_image_trash(pool, image):
+            log.info("Not able to move the image to trash as device or resource busy")
```

Comment:
Can we change this message to a more relevant one for the test case, something like

"Image not moved to trash since it is in secondary cluster"